src/Agent.Sdk/Util/StringUtil.cs

```diff
@@ -181,6 +181,20 @@ private static string Format(CultureInfo culture, string format, params object[]
             }
         }
 
+        // Used for L1 testing
+        public static void LoadExternalLocalization(string stringsPath)
```

Comment:
Personal preference would be to have a class like `TestingStringUtil` in the test project with this method instead of adding it to non testing code if we only need it for testing.

Having it here implies it's got production quality tests and some use outside of testing.

Reply:
On second look we are setting some values on the util here.

This setup feels wrong to me :) It may be out of the scope of the PR but feels like it's structured incorrectly.

Maybe also this class should be split into 2? One for `StringUtil` and one for `LocalizationStringUtil`?

Reply:
This class might be worth revisiting in the future, but it seems out of scope here. With the current design, this seems like the best way to achieve the objective of getting some localization loaded for testing.